pingrid.py

```diff
@@ -448,13 +452,20 @@ def parse_colormap(s: str) -> np.ndarray:
     #     len(vs),
     #     [f"{v.red:02x}{v.green:02x}{v.blue:02x}{v.alpha:02x}" for v in vs],
     # )
-    rs = np.array([vs[int(i / 256.0 * len(vs))] for i in range(0, 256)], np.uint8)
+    if thresholds is None:
+        rs = np.array([vs[int(i / 256.0 * len(vs))] for i in range(0, 256)], np.uint8)
+    else:
+        rs = np.full([256, 4], np.nan)
+        for i in range(0, thresholds.size):
+            rs[int(255.*(thresholds[i]-thresholds[0])/(thresholds[-1]-thresholds[0])), :] = np.array([vs[i]], np.uint8)
+        dfrs = pd.DataFrame(rs)
+        rs = dfrs.interpolate().astype(int).values
     return rs
```

Comment:
Is this the "colorscale business" you're saying I approved in the pycpt maproom PR? It doesn't look familiar to me.

Seems ok, but this is a note to myself to verify that it doesn't break fbfmaproom before I approve.

Reply:
That and more. But it may be inaccurate to claim you have approved. See commits
[pingrid can now make colormaps unequally spaced](https://github.com/kjhall-iri/pycpt-remic/pull/7/commits/6f6f49ce8fe1ab190a0b69dfabe41212591c1941)
to
[the right static colorscale](https://github.com/kjhall-iri/pycpt-remic/pull/7/commits/c01311bf2a60fb18b85a886b4a84bb7822efd74b)
from here:
https://github.com/kjhall-iri/pycpt-remic/pull/7/commits